<launch>
  <!-- you can load an exisiting tree with <node ... args="tree.bt"> !-->
  <node pkg="octomap_server" type="octomap_tracking_server_node" name="octomap_talker" output="screen">
    <param name="resolution" value="0.02" />
    <param name="frame_id" type="string" value="base_link" />
    <param name="sensor_model/max_range" value="4.0" />
    <param name="save_directory" value="$(optenv OCTOMAP_SAVE_DIR ./)" />
    <param name="track_changes" value="true"/>
    <param name="listen_changes" value="false"/>
    <param name="topic_changes" value="/octomap_tracking_server/changeset" />
    <remap from="cloud_in" to="/laser/point_cloud" />
  </node>
  
  <node pkg="tf" type="static_transform_publisher" name="world2map" args="0 0 0 0 0 0  world map 100" />

</launch>
